QLogger.h

```diff
@@ -156,7 +256,7 @@ void QLog_(const QString &module, QLogger::LogLevel level, const QString &messag
  */
 #   define QLog_Trace(module, message)                                                                                 \
       QLogger::QLoggerManager::getInstance()->enqueueMessage(module, QLogger::LogLevel::Trace, message, __FILE__,      \
-                                                             __LINE__)
+                                                             __LINE__, __FUNCTION__)
```

Comment:
I get a compilation error in here. __FUNCTION__ should be before __FILE__, but I can fix that after merge.

Reply:
Oh, sorry it's a mistake